index.bs

```diff
@@ -815,6 +815,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=boolean=].
 : <dfn>fenced</dfn>
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/reporting origin=] per [=attribution source/attribution destinations|attribution destination=].
```

Comment:
Why does this say `per reporting origin`? The source is only associated with 1 reporting origin.

Reply:
similar to `max_event_states`, `attribution_scope_limit` is per reporting origin per destination. it's not associated with the source directly specifically.

Reply:
But an individual source only has one reporting origin. So this limit is applied per destination for the source's reporting origin, no?

Reply:
good point, will update the documentation to be more precise.